tests/test_docker_builder.py

```diff
@@ -42,6 +43,7 @@ def test_color():
 
 
 EXPECTED_GPU_DOCKERFILE = f"""# Change CUDA and cuDNN version here
+ARG PYTHON_VERSION=3.12
```

Comment:
do you want to add a test case checking that also passed arguments are respected here?